src/Api/JsonRpc/Client.php

```diff
@@ -387,46 +387,43 @@ public function addTestResult($name, $result, $containingClass, $text = "", $exe
         }
     }
 
-    public function createTestExecution(int $testCaseId, int $statusId): TestExecution
+    public function createTestExecution(int $testCaseId, int $statusId): array
     {
         $testExecution = new TestExecution();
         $testExecution->setRunId($this->testRun->getRunId());
         $testExecution->setCaseId($testCaseId);
         $testExecution->setBuildId($this->testRun->getBuildId());
+        $testExecution->setStatusId($statusId);
```

Comment:
I don't understand why you need to set all of these values when `testExecutionRepository->create()` only needs run_id and case_id values, see https://github.com/kiwitcms/phpunit-plugin/pull/37/files#diff-21a369f3d498099fcd2a2c62d1411cbca390330e4aa8c7059fb4adb03194f2aeR68. 

Why not pass the 2 id values directly instead of creating some sort of intermediary object ?

Reply:
I guess the build is redundant, before that this method do 2 things: first creates the TestExecution and after that updates its status. So I want to reduce the api calls and create the TestExecution with the correct status.

Reply:
well you can't reduce them because these are 2 different API methods. `TestRun.add_case()` and `TestExecution.update()`

Reply:
Yes, my bad